src/main/java/no/nav/familie/integrasjoner/saksbehandler/SaksbehandlerService.kt

```diff
@@ -11,6 +12,13 @@ class SaksbehandlerService(private val azureGraphRestClient: AzureGraphRestClien
 
     fun hentSaksbehandler(id: String): Saksbehandler {
 
+        if (id == ID_VEDTAKSLØSNINGEN) {
```

Comment:
Føles som ett unødvendig kall fra klienten (familie-tilbake?), er det kanskje bedre å stoppe det i klienten i stedet?

Reply:
Jeg tenker dette er mer robust med tanke på bruk fra andre løsninger. Hvis vi skal ha automatisert saksbehandling på sikt, så bør "VL" ha et navn som medfører an felles signatur for ytelsene.